AWSClientRuntime/Package.swift

```diff
@@ -42,11 +42,11 @@ if let smithySwiftDir = ProcessInfo.processInfo.environment["SMITHY_SWIFT_CI_DIR
    let crtDir = ProcessInfo.processInfo.environment["AWS_CRT_SWIFT_CI_DIR"] {
     package.dependencies += [
         .package(name: "AwsCrt", path: "\(crtDir)"),
-        .package(name: "ClientRuntime", path: "\(smithySwiftDir)/Packages")
+        .package(name: "ClientRuntime", path: "\(smithySwiftDir)")
```

Comment:
nit: string interpolation is not needed at most `path` params.
```suggestion
        .package(name: "ClientRuntime", path: smithySwiftDir)
```

Reply:
Fixed this and several other places where interpolation is unneeded